pkg/vcs/vcs_test.go

```diff
@@ -155,3 +157,35 @@ func TestCommitLink(t *testing.T) {
 		}
 	}
 }
+
+func TestParse(t *testing.T) {
+	lines := []string{"Masahiro Yamada <masahiroy@kernel.org> " +
+		"(maintainer:KERNEL BUILD + files below scripts/ (unless mai...)",
+		"Michal Marek <michal.lkml@markovi.net>" +
+			"(maintainer:KERNEL BUILD + files below scripts/ (unless mai...)",
+		"linux-kbuild@vger.kernel.org" +
+			"(open list:KERNEL BUILD + files below scripts/ (unless mai...)",
+		"linux-kernel@vger.kernel.org (open list)"}
+	maintainers := []RecipientInfo{{"masahiroy@kernel.org",
+		"Masahiro Yamada",
+		"(maintainer:KERNEL BUILD + files below scripts/ (unless mai...)",
+		true},
+		{"michal.lkml@markovi.net",
```

Comment:
We try to not hardcode real people names/emails in the code base. They may not like it or complain. We can get the same testing with artificial names/emails.